### PR TITLE
Added button to copy jobs info to clipboard

### LINKED ISSUE #909: Add a copy to clipboard button for the list of jobs
Sometimes it's necessary to copy all URL -addresses. It'll be good to have a corresponding button somewhere near the list of jobs.
copy_jobs_urls

### PR DESCRIPTION
**Pull request by bsekachev**
Resolved #909

Submitted codacy issue: https://github.com/codacy/codacy-eslint/issues/153

### CODE REVIEW

File: cvat-ui/src/components/cvat-app.tsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ export default class CVATApplication extends React.PureComponent<CVATAppProps> {
                     />
                 ),
                 duration: null,
-                description: error.length > 200 ? '' : error,
+                description: error.length > 200 ? 'Open console to details (F12, Console tab)' : error,
```
Comment (nmanovic) [inline cvat-ui/src/components/cvat-app.tsx]: F12 doesn't work for Safari.

File: cvat-ui/src/utils/copy-to-clipboard.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default function copyToClipboard(str: string): void {
```
Comment (nmanovic) [inline cvat-ui/src/utils/copy-to-clipboard.ts]: Let's use the library instead https://github.com/sudodoki/copy-to-clipboard/

File: cvat-ui/src/components/cvat-app.tsx [hunk 3]
```diff
@@ -139,7 +139,7 @@ export default class CVATApplication extends React.PureComponent<CVATAppProps> {
                     />
                 ),
                 duration: null,
-                description: error.length > 200 ? '' : error,
+                description: error.length > 200 ? 'Open browser console to get details' : error,
```
Comment (nmanovic) [inline cvat-ui/src/components/cvat-app.tsx]: Need to have an article. Let's use `your` instead.
```suggestion
                description: error.length > 200 ? 'Open your browser console to get details' : error,
```

File: cvat-ui/src/utils/copy-to-clipboard.ts (hunk 2 above)
Comment (bsekachev) [reply cvat-ui/src/utils/copy-to-clipboard.ts]: Done

File: cvat-ui/src/components/cvat-app.tsx (hunk 3 above)
Comment (bsekachev) [reply cvat-ui/src/components/cvat-app.tsx]: Done

**Pull request merged**